python/lsst/ts/MTMount/mock/controller.py

```diff
@@ -68,6 +76,29 @@ def make_reply_dict(id, **parameters):
     )
 
 
+def as_scalar_or_list(value, nelts):
+    """Return a value as a list or scalar, depending on nelts.
+
+    Parameters
+    ----------
+    value : typing.Any
+        Value to return
+    nelts : int
+        Number of items. If 1 then return ``value`` as a scalar.
```

Comment:
Perhaps change to `Number of elements`? You seem to be using `elements` everywhere and not `items`.

Reply:
I forgot to do this before merging, but have saved the change for the next ticket.